admin/section/class-convertkit-settings-general.php

```diff
@@ -166,7 +166,29 @@ public function register_fields() {
 			$this->settings_key,
 			$this->name
 		);
+		add_settings_field(
+			'clear_logs',
+			'',
+			array( $this, 'clear_logs' ),
+			$this->settings_key,
+			$this->name
+		);
+
+		add_settings_field(
+			'debug',
+			'Debug',
+			array( $this, 'debug_callback' ),
+			$this->settings_key,
+			$this->name
+		);
 
+		add_settings_field(
+			'debug',
+			'Debug',
+			array( $this, 'debug_callback' ),
+			$this->settings_key,
+			$this->name
+		);
```

Comment:
It looks like this is called the same way three times. Are they all needed? https://github.com/ConvertKit/convertkit-wordpress/blob/211a0b3fe6b9cfc522e940d0fef9248cebb38150/admin/section/class-convertkit-settings-general.php#L177-L198

Reply:
This was a mistake, this should be fixed now.